app/presenters/security_check_failed_presenter.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class SecurityCheckFailedPresenter
+  def troubleshooting_options
+    [
+      BlockLinkComponent.new(
+        url: MarketingSite.contact_url,
```

Comment:
We should use the redirect controller to make sure we log when the link is clicked. Interestingly, we have a [redirect controller dedicated to this URL](https://github.com/18F/identity-idp/blob/main/app/controllers/redirect/contact_controller.rb), but we don't seem to use it anywhere. 🤔 🤷 

```suggestion
        url: contact_redirect_url,
```

Reply:
didnt even know it existed